src/Test.EndToEnd.Baselining/Options/DebugOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using CommandLine;
+
+namespace Test.EndToEnd.Baselining.Options
+{
+    [Verb("debug", HelpText = "Debug Baselining for a specific Series, Log, and Result")]
+    public class DebugOptions : OptionsBase
+    {
+        [Value(1, Required = false, HelpText = "Debug: Series Path (everything under 'Input/'")]
+        public string DebugSeriesPath { get; set; }
+
+        [Value(2, Required = false, HelpText = "Debug: Log Index")]
+        public int DebugLogIndex { get; set; }
```

Comment:
Since values 2 and 3 are not required, we need to document their defaults as well.

Reply:
Fixed

Reply:
Fixed.